### PR TITLE
Add documentation for ZigBee ZoneStatus

### PR DESCRIPTION
**Pull request by varzac**
@tpmanley @workingmonk @jimmyjames

### CODE REVIEW

File: ref-docs/zigbee-ref.rst [hunk 1]
```diff
@@ -407,3 +407,105 @@ Returns true if the compared hex values are not equal.
...
+
+Additional ZigBee Classes
+-------------------------
+
+There are some additional classes that are provided to make interacting with and handling Zigbee messages.
+
+ZoneStatus
+^^^^^^^^^^
+
+The ZoneStatus class is made to help with parsing and giving a simple interface for dealing with responses
```
Comment (tpmanley) [inline ref-docs/zigbee-ref.rst]: Might be more clear to just say "for dealing with the ZoneStatus attribute", and also add " in the IAS Zone cluster".

**Comment by tpmanley** [comment]
👍

File: ref-docs/zigbee-ref.rst (hunk 1 above)
Comment (unixbeast) [inline ref-docs/zigbee-ref.rst]: incomplete sentence. Might want to say "...to make interacting with and handling ZigBee messages easier." or something along those lines.

File: ref-docs/zigbee-ref.rst [hunk 2]
```diff
@@ -407,3 +407,119 @@ Returns true if the compared hex values are not equal.
...
+
+**Parameters:**
+    - **description**: A zone status message description.
+
+
+
+Additional ZigBee Classes
+-------------------------
+
+There are some additional classes that are provided to make interacting with and handling Zigbee messages.
```
Comment (unixbeast) [inline ref-docs/zigbee-ref.rst]: incomplete sentence. Might want to say "...to make interacting with and handling ZigBee messages easier." or something along those lines.

File: ref-docs/zigbee-ref.rst [hunk 3]
```diff
@@ -407,3 +407,119 @@ Returns true if the compared hex values are not equal.
...
+
+ZoneStatus
+^^^^^^^^^^
+
+The ZoneStatus class is made for dealing with the ZoneStatus attribute in the IAS Zone cluster.  It has a
+single constructor that takes an int which is the ZoneStatus attribute value.
```
Comment (unixbeast) [inline ref-docs/zigbee-ref.rst]: reword a bit:
"The purpose of the ZoneStatus class is to handle the ZoneStatus attribute in the IAS Zone cluster. It has a single constructor that takes an `int` parameter that represents the ZoneStatus attribute value."

File: ref-docs/zigbee-ref.rst [hunk 4]
```diff
@@ -407,3 +407,119 @@ Returns true if the compared hex values are not equal.
...
+                                  | 0 - OK
+7           ac                    | 1 - ac/mains fault
+                                  | 0 - ac/mains OK
+8           test                  | 1 - sensor is in test mode
+                                  | 0 - sensor is in operation mode
+9           batteryDefect         | 1 - sensor detects a defective battery
+                                  | 0 - sensor battery is functioning normally
+========== ===================== ===========================================
+
+See the ZigBee Home Automation specification and the ZCL specification for more information.
```
Comment (unixbeast) [inline ref-docs/zigbee-ref.rst]: Can we make these links like we do in the ZigBee primer?
`ZigBee Cluster Library (ZCL) <http://www.zigbee.org/download/standards-zigbee-cluster-library/>`__

`ZigBee Home Automation (HA) <http://www.zigbee.org/zigbee-for-developers/applicationstandards/zigbeehomeautomation/>`__

**Comment by unixbeast** [comment]
Thanks for the docs addition Zach! I've added a few minor inline comments.

**Pull request merged**